README.md

```diff
@@ -92,60 +95,96 @@ pointed to that branch. Please, make sure you follow the
 require that at least two developers review them first before merging to `dev`
 branch.
 
+### Installing RIOT-OS
+
+RIOT is a real-time multi-threading operating system that supports a range of devices that are typically found in the Internet of Things (IoT): 8-bit, 16-bit and 32-bit microcontrollers.
+
+### Pre-requisites
+
+1. Install Git.
+2. Install the build-essential packet. This varies based on the operating system in use.for `mac` we recommend using` xcode-select install`.
+3. Install OpenOCD
+4. Install  python-serial python3 python3-flake8
+5. install coccinelle
+6. install curl
+7. install make
+8. install doxygen
+9. install unzip 
+10. install wget
+ 
+ if you use MacOS we recommend you use `brew`.
+ 
 ### Installing ESP-IDF
 
-In order to compile the firmware (and to flash it) you need to install the
-[ESP-IDF](https://github.com/espressif/esp-idf/) development framework, which
-brings the compiler and necessary tools in order to correctly build the
-project.
+ESP-IDF, the official SDK from Espressif, can be downloaded and installed as GIT repository.
+1) create folder in $HOME called `esp`
+1) cd $HOME/esp
+2) git clone https://github.com/espressif/esp-idf.git
+3) cd esp-idf
+4) git checkout f198339ec09e90666150672884535802304d23ec
+5) git submodule init
+6) git submodule update
 
-Please note that we use a fork of the ESP-IDF, that enables us to use SLIP as
-a network interface (SLIP ESP-NETIF) until we get upstream support, it can
-be used as follows:
 
+### Install Manual toolchain
+
+The quickest way to start development with ESP32 is by installing a prebuilt toolchain. Below you will be given the installation steps for the toolchain by operating system:
+
+1) [MacOS](https://docs.espressif.com/projects/esp-idf/en/release-v3.0/get-started/macos-setup.html)
+
+2) [Linux](https://docs.espressif.com/projects/esp-idf/en/release-v3.0/get-started/linux-setup.html)
```

Comment:
Recommend using the RIOT toolchain for ESP32.

See https://doc.riot-os.org/group__cpu__esp32.html#esp32_toc

Also, what do you think of recommending docker for Mac OS X users?